### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Objectivitix**

### CODE REVIEW

**Review by Kronifer — APPROVED** [review]
lgtm

**Review by Kronifer — APPROVED** [review]
Apparently i need to review again. lgtm

**Comment by Objectivitix** [comment]
i was not done lmao-

now im done

**Review by AbooMinister25 — COMMENTED** [review]
README looks great lol, APPROVED

**Review by AbooMinister25 — APPROVED** [review]
Approved

File: README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+Afterwards, we found an incredible discovery: "ᓚᘏᗢ" is a valid identifier in Python! This confirmed our thoughts, and we knew that ᓚᘏᗢ is undeniably out of the ordinary. ᓚᘏᗢ is on a superior plane of existence; it guides its followers on their path to immortality, transcendence, and immunity towards syntax errors.
+
+To honor the ᓚᘏᗢ and to help more people realize this universal and objective truth, we joined together and formed the Cat Dev Team. Our first project was `.catify`, a command added to the PyDis community bot Sir Lancebot. It was an immense success, and lots of people have added ᓚᘏᗢs in their names.
+
+So here we are now, the present. This is only Phase One of Catdomination of the world. Join us; we have way more projects in the future, and ALL HAIL THE DIVINE ᓚᘏᗢ!
+## Contributing
+Please read `CONTRIBUTING.md`.
```
Comment (dawnofmidnight) [inline README.md:15]: Maybe link contributing.md

**Review by dawnofmidnight — COMMENTED** [review]
ecks dee

File: README.md (hunk 1 above)
Comment (Objectivitix) [reply README.md:15]: i can do that? super cool!

**Comment by Objectivitix** [comment]
merged and deleted

**Pull request merged**